evm-template/runtime/src/apis.rs

```diff
@@ -503,18 +503,111 @@ impl_runtime_apis! {
                 }
 
                 fn reserve_transferable_asset_and_dest() -> Option<(Asset, Location)> {
+                    use xcm_primitives::AssetTypeGetter;
+                    use frame_system::RawOrigin;
+
+                    // set up fee asset
+                    let fee_location = RelayLocation::get();
+                    let who: AccountId = frame_benchmarking::whitelisted_caller();
+
+                    let Some(location_v3) = xcm::v3::Location::try_from(fee_location.clone()).ok() else {
+                        return None;
+                    };
+                    let asset_type = AssetType::Xcm(location_v3);
+
+                    let local_asset_id: crate::types::AssetId = asset_type.clone().into();
+                    let manager_id = AssetManager::account_id();
+                    let _ = Assets::force_create(RuntimeOrigin::root(), local_asset_id.clone().into(), sp_runtime::MultiAddress::Id(manager_id.clone()), true, 1);
+                    let _ = Assets::mint(
+                        RawOrigin::Signed(manager_id.clone()).into(),
+                        local_asset_id.into(),
+                        sp_runtime::MultiAddress::Id(who),
+                        3001010000000,
```

Comment:
nitpick: usually a better idea to use the magic number as a constant declared with an explanatory name

Reply:
Moved to a constant type